Modules/Office365DSC/Modules/O365DSCLogEngine.psm1

```diff
@@ -0,0 +1,40 @@
+<# This method creates a new error log file for each session,
+   whenever an error is encountered, and appends valuable
+   troubleshooting information to the file;
+#>
+function New-Office365DSCLogEntry
+{
+    [CmdletBinding()]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [System.Object]
+        $Error,
+
+        [Parameter()]
+        [System.String]
+        $Message
+    )
+
+    # Get the current folder we are executing the script from;
+    $ExecPath = $SCRIPT:MyInvocation.MyCommand.Path
+    $LastSlash = $ExecPath.ToString().LastIndexOf("\")
```

Comment:
Isn't "Split-Path -Path $ExecPath -Parent" easier in this case?

Reply:
You are right, and actually, because I am trying to save the file within the current execution folder, I don't even need to determine this. Out-File will save in current folder by default. So I completely removed these lines.